config/routes.rb

```diff
@@ -325,10 +325,10 @@
   get 'questions_shadow' => 'questions#index_shadow'
   get 'question' => 'questions#index'
   get 'question_shadow' => 'questions#index_shadow'
-  get 'questions/:author/:date/:id' => 'questions#show' 
+  get 'questions/:author/:date/:id' => 'questions#show'
   get 'questions/show/:id' => 'questions#show'
   get 'q/:id' => 'questions#shortlink'
-  get 'questions/answered(/:tagnames)' => 'questions#answered'
+  get 'questions/recently_commented(/:tagnames)' => 'questions#recently_commented'
```

Comment:
Hmm. Here, should we preserve the old route, redirecting it to the new route? That way we don't break any existing URLs people have made out there? 

I think it's: `get '/stories', to: redirect('/posts')` for example? 

Thank you!!

Reply:
i have made the changes.. i have attached a gif showing the redirection happening. Thanx

Reply:
Oh great -- however i don't see the routes changes here... did you push them up? Thanks!

Reply:
```suggestion
  get 'questions/answered(/:tagnames)', to: redirect('questions#recently_commented')
  get 'questions/recently_commented(/:tagnames)' => 'questions#recently_commented'
```

That should work!